src/logic/groups.js

```diff
@@ -429,6 +436,10 @@ const assignScrambling = (wcif, roundsToAssign) => {
             : sortByArray(available(competitors), competitor => [
                 /* Optionally avoid assigning tasks to newcomers. */
                 noTasksForNewcomers && !competitor.wcaId ? 1 : -1,
+                tasksForOwnEventsOnly &&
```

Comment:
Just for completeness.

```suggestion
                /* Optionally avoid assigning tasks in events the person didn't register for. */
                tasksForOwnEventsOnly &&
```